### ISSUE TITLE
How to use the compiled Release package (2016-3-10)?

### ISSUE THREAD

**Issue by hsl9999** [DigitalRune/DigitalRune#10]
DigitalRune Engine 2016-03-10 (For existing customer only!)
Could I have a license key that I can use?
I wanted to simplify the process to test some of my applications.
Thanks a lot.

**Comment by luxbinary** [comment]
I dont think you will get a licence key for that. It was just the download for existing customer and the repo it not maintained anymore.
Why dont you just compile it by yourself?
There are details instructions at https://github.com/DigitalRune/DigitalRune/issues/4#issuecomment-238166396

**Comment by hsl9999** [comment]
Thank you for your reply.
I will try it.
